sdk/tables/azure-data-tables/inc/azure/data/tables/constants.hpp

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+#pragma once
+
+namespace Azure { namespace Data { namespace Tables { namespace _detail {
```

Comment:
This can now move to `/src/private` and reduce visibility to end customers since it is only referenced by `tables_clients.cpp`.

I was planning to do that in https://github.com/Azure/azure-sdk-for-cpp/pull/6135